DemoServer/AdditionalLanguages/python/basics/createDocument/metadata.json

```diff
@@ -0,0 +1,112 @@
+                "  Collections are used to group documents together so that it is convenient to apply some operation to them.",
+                "",
+                "* _The Python class members_ determine the document JSON structure in the database.<br>",
+                "  The JSON serialization for the document fields follows the conventions set on the Document Store",
+                "",
+                "* Note: Having an 'Id' field property in the class definition is Not mandatory.<br>",
+                "  The Session will assign an ID for the entity that is stored, and this ID can be fetched later at any time.",
+                "  See step 3 for more details."
+            ],
+            "Assets": [
+                {
+                    "Title": "Document Modeling",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/3-document-modeling#document-modeling"
+                }
+            ]
+        },
+        {
+            "Title": "Store the new company object in the Session",
+            "Slug" : "step-2",
+            "DescriptionLines": [
+                "* Associate the new entity with the Session `store()` command.",
+                "",
+                "* Note: The new document will be persisted to the database once _save_changes()_ is called.<br>",
+                "  See step-4.",
+                "",
+                "* However, the store() command will _immediately_ assign an ID to the new entity,<br>",
+                "  so you can refer to it in other documents that you'll save in the same batch.<br>",
+                "  See how to get this ID in the next step."
+            ],
+            "Assets": [
+                {
+                    "Title": "The Store Command",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/2-zero-to-ravendb#store"
+                },
+                {
+                    "Title": "Storing Entities",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/java/client-api/session/storing-entities"
+                }
+            ]
+        },
+        {
+            "Title": "The new document ID is immediately available",
+            "Slug" : "step-3",
+            "DescriptionLines": [
+                "* The Session takes care of generating the document identifier automatically.<br>",
+                "  The new stored entity ID is now available and can be used.",
+                "",
+                "* _If your Python class was defined with an 'Id' property_:<br>",
+                "  => You can get the new stored entity ID as shown in the above example.",
+                "",
+                "* _If your Python class was Not defined with an 'Id' property_:<br>",
```

Comment:
even when we don't specify 'Id' field in the class definition, the client will add such a field to the object, taking leverage of Python dynamic typing

Reply:
done